modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/modals/CancelEditAPIApplicationModalContent.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import React from 'react';
+
+export function CancelEditAPIApplicationModalContent({
```

Comment:
@cgmonte I think you could name this component as "CancelEditAPIApplicationModal", since you're not defining just the content but the whole modal itself.

Reply:
Ok, we can keep this pattern then :)